src/app/js/fields/wizard/index.js

```diff
@@ -132,7 +132,9 @@ class FieldEditionWizardComponent extends Component {
                     key="transformers"
                     isSubresourceField={!!field.subresourceId}
                 />,
-                !field.subresourceId && <StepSemantics key="semantics" />,
+                !field.subresourceId && (
+                    <StepSemantics fields={fields} key="semantics" />
```

Comment:
You add a prop without changing the `StepSemantics` component or its propTypes, was it already there?

Reply:
it was a missing prop
maybe not fully necessary, not checked because this PR is not the goal, but surely asked by proptypes